combined.js

```diff
@@ -0,0 +1,159 @@
+// ICEBOX - gives user the option to play either game
+
+const { reverse } = require("dns");
+const readline = require("readline");
+const { start } = require("repl");
+const rl = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+  return new Promise((resolve, reject) => {
+    rl.question(questionText, resolve);
+  });
+}
+
```

Comment:
Hoisting only works for a certain type of function (declaration) and not the other (expression). While it is viable to hoist your function declaration invocation (as you did here), I would caution you against utilizing it until you are further along in your knowledge of Javascript and the choice is more informed.